docs/apis-tools/operate-api/authentication.md

```diff
@@ -63,11 +147,14 @@ Another way to access the Operate API in a Self-Managed cluster is to send cooki
 1. Log in as user 'demo' and store the cookie in the file `cookie.txt`.
 
 ```shell
-curl -c cookie.txt -X POST 'http://localhost:8080/api/login?username=demo&password=demo'
+curl --request POST 'http://localhost:8080/api/login?username=demo&password=demo' \
+   --cookie-jar cookie.txt
 ```
 
 2. Send the cookie (as a header) in each API request. In this case, request all process definitions.
```

Comment:
NBC: Does it need brackets?

Send the cookie as a header in each...

Reply:
```suggestion
2. Send the cookie as a header in each API request. In this case, request all process definitions.
```